### DISCUSSION TITLE
UpSnap wol only few seconds after turning off a pc

### DISCUSSION THREAD

**Discussion by lucasca-maker** [seriousm4x/UpSnap#838] [Help wanted]
Hello all!
I installed UpSnap on my qnap TS-431k nas and I would like to wol the pc's connected to the nas.
My problem is that it turns on the pc's only if they have been recelently (10 seconds) turned off. I can always see the state of the pc's (if the are on or off) from UpSnap

Thanks for the help
Luca

P.S.
if I try to send the magic packet from another pc inside the same network it akways work

**Comment by seriousm4x** [comment]
That sounds like some power management configuration on those pc's that send the NICs to sleep after 10 seconds. I don't think that I can solve this in my code. Also there is no way I can reproduce this in my network.
